x/wasm/client/testutil/cli_test.go

```diff
@@ -1,9 +1,13 @@
+//go:build norace
+// +build norace
```

Comment:
Why do we need to add a norace tag?

Reply:
Added this tag to avoid redundant test. If we don't add this tag, it execute twice